woocommerce-admin.php

```diff
@@ -98,6 +98,7 @@ function wc_admin_build_file_exists() {
  */
 function wc_admin_do_wc_admin_daily() {
 	WC_Admin_Notes_New_Sales_Record::possibly_add_sales_record_note();
+	WC_Admin_Notes_Mobile_App::possibly_add_mobile_app_note();
```

Comment:
Should we make this a filterable list?

Reply:
I think the `wc_admin_daily' action (see a couple lines down) covers the extendability usecase for us. These two methods can be unhooked, and new ones can be hooked in.